### DISCUSSION TITLE
RSS feed: Adjust number of entries

### DISCUSSION THREAD

**Discussion by PeterBenzin** [nextcloud/bookmarks#1971] [Ideas]
Hi! Is it already possible to adjust how many entries the RSS feed has? Default seems to be 10. Otherwise this would be highly appreciated!

**Comment by marcelklehr** [comment]
See https://nextcloud-bookmarks.readthedocs.io/en/latest/bookmark.html#query-bookmarks for documentation of the bookmark endpoint

**Reply by PeterBenzin** [reply]
Thank you - very helpful! :)
